src/main/java/org/jvnet/hudson/test/MemoryAssert.java

```diff
@@ -146,13 +148,16 @@ public static void assertGC(WeakReference<?> reference) {
     }
 
     /**
+     * <strong>Assumes Java runtime is &le; Java 8. I.e. tests will be skipped if Java 9+</strong>
      * Forces GC by causing an OOM and then verifies the given {@link WeakReference} has been garbage collected.
      * @param reference object used to verify garbage collection.
      * @param allowSoft if true, pass even if {@link SoftReference}s apparently needed to be cleared by forcing an {@link OutOfMemoryError};
      *                  if false, fail in such a case (though the failure will be slow)
      */
     @SuppressWarnings("DLS_DEAD_LOCAL_STORE_OF_NULL")
     public static void assertGC(WeakReference<?> reference, boolean allowSoft) {
+        // Disabled on Java 9+, because below will call Netbeans Insane Engine, which in turns tries to call setAccessible
+        assumeTrue(System.getProperty("java.specification.version").startsWith("1."));
```

Comment:
Comment says Java 9 but assertion does not mention 8.  Would it be worth considering:
```suggestion
        assumeTrue(System.getProperty("java.specification.version").startsWith("1.8"));
```